src/components/pathway/SearchPathwayCard.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { useContext, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import Truncate from 'react-truncate';
+import { Link } from 'react-router-dom';
+import Skeleton from 'react-loading-skeleton';
+import { AppContext } from '@edx/frontend-platform/react';
+import { getConfig } from '@edx/frontend-platform/config';
+import { camelCaseObject } from '@edx/frontend-platform/utils';
+import { Badge, Card } from '@edx/paragon';
+import { sendEnterpriseTrackEvent } from '@edx/frontend-enterprise-utils';
+import classNames from 'classnames';
+
+const filterTags = tags => {
+  const singlTagSizeLimit = 45;
+  const offset = 5;
+  let limit = 90;
+  const tagsToReturn = [];
+  for (let i = 0; i < tags.length; i++) {
+    const tag = tags[i];
+
+    if (tag.length < singlTagSizeLimit && tag.length <= limit) {
+      tagsToReturn.push(tag);
+
+      limit -= (tag.length + offset);
+    }
+  }
+
+  return tagsToReturn;
+};
+
+const SearchPathwayCard = ({ hit, isLoading }) => {
+  const { enterpriseConfig: { slug, uuid: enterpriseCustomerUUID } } = useContext(AppContext);
+
+  const pathway = hit ? camelCaseObject(hit) : {};
+
+  const linkToPathway = useMemo(
```

Comment:
I think learner pathways will not have separate page?

Reply:
I have updated the link to not point to a new page.